andela/pulse-mobile-app
# [#156619140] Add app icon

### What does this PR do?
Updates App Icon from the default one to the Andela Logo


### How should this be manually tested?
 -  run `react-native run-android` and `react-native run-ios` to check icons for android and ios respectively.
- navigate to simulator / phone menu to view the App Icon in app listings

### What are the relevant pivotal tracker stories?
[#156619140](https://www.pivotaltracker.com/story/show/156619140)

### Screenshots (if appropriate)
![screen shot 2018-04-12 at 19 09 03](https://user-images.githubusercontent.com/17447937/38689781-2485e8ec-3e85-11e8-9f11-f53c4d1e1f76.png)
![screen shot 2018-04-12 at 19 10 54](https://user-images.githubusercontent.com/17447937/38689831-41912672-3e85-11e8-883c-f2bc46265998.png)
